### ISSUE TITLE
Recent update gives a 500 error after login

### ISSUE THREAD

**Issue by deljones** [YunoHost-Apps/wallabag2_ynh#140]
### Describe the bug

500 error after login

### Context

- Hardware: *VPS bought online / Old laptop or computer / Raspberry Pi at home / Internet Cube with VPN / Other ARM board / ...*
- YunoHost version: 4.3.6.3
- I have access to my server: *Through SSH | through the webadmin | direct access via keyboard / screen | ...*
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: *no / yes*
  - If yes, please explain:
- Using, or trying to install package version/branch:
- If upgrading, current package version: *can be found in the admin, or with `yunohost app info $app_id`*

### Steps to reproduce

Updated to the current version of Wallabag. 
Enter the URL for the application.
Login screen appears.
Enter login details.
The following error appears:

```
Oops! An Error Occurred
The server returned a "500 Internal Server Error".
Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.
```

### Expected behavior

Should go to the articles saved

### Logs
Cant find the log files... Help!

**Comment by Schdefoon** [comment]
Hi,

same problem for me. I found the logfile in /var/www/wallabag2/var/logs/prod.log
But mine has grown to 1.5 GB in three years, so it is not very easy to get the values inside,,,

But my Wallabag-database is still filled with new entries if I send them to Wallabag - so there's hope that it is only a minor problem

Best regards

**Comment by Schdefoon** [comment]
I think this is the part of the logfile that is written in the logfile when trying to open the Wallabag main site:

`[2022-07-24 14:10:36] request.INFO: Matched route "unread". {"route":"unread","route_parameters":{"page":"1","_controller":"Wallabag\\CoreBundle\\Controller\\EntryController::showUnreadAction","_route":"unread"},"request_uri":"https://yunohost.local/wallabag/unread/list","method":"GET"} []
[2022-07-24 14:10:36] security.INFO: Basic authentication Authorization header found for user. {"username":"MyUserName"} []
[2022-07-24 14:10:36] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Ldap\Exception\LdapException: "The LDAP PHP extension is not enabled." at /var/www/wallabag2/vendor/symfony/symfony/src/Symfony/Component/Ldap/Adapter/ExtLdap/Adapter.php line 29 {"exception":"[object] (Symfony\\Component\\Ldap\\Exception\\LdapException(code: 0): The LDAP PHP extension is not enabled. at /var/www/wallabag2/vendor/symfony/symfony/src/Symfony/Component/Ldap/Adapter/ExtLdap/Adapter.php:29)"} []
[2022-07-24 14:40:58] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"Wallabag\\CoreBundle\\Controller\\EntryController::showUnreadAction","page":1,"_route":"homepage"},"request_uri":"https://yunohost.local/wallabag/","method":"GET"} []
[2022-07-24 14:40:58] security.INFO: Basic authentication Authorization header found for user. {"username":"MyUserName"} []
[2022-07-24 14:40:58] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Ldap\Exception\LdapException: "The LDAP PHP extension is not enabled." at /var/www/wallabag2/vendor/symfony/symfony/src/Symfony/Component/Ldap/Adapter/ExtLdap/Adapter.php line 29 {"exception":"[object] (Symfony\\Component\\Ldap\\Exception\\LdapException(code: 0): The LDAP PHP extension is not enabled. at /var/www/wallabag2/vendor/symfony/symfony/src/Symfony/Component/Ldap/Adapter/ExtLdap/Adapter.php:29)"} []`

**Comment by deljones** [comment]
yes the mobile app works fine, so I can get in and out no problem.

Dj

**Comment by JimboJoe** [comment]
same error on a private session in the navigator, but different error on an existing one navigator session:
```
500: Internal Server Error

An exception has been thrown during the rendering of a template ("The Symfony\Component\Intl\DateFormatter\IntlDateFormatter::__construct() method's argument $locale value 'en_EN' behavior is not implemented. Only the locale "en" is supported. Please install the "intl" extension for full localization capabilities.").
```
error 500 also with Firefox extension or Android app...
```
[2022-07-24 17:26:27] graby.INFO: Attempting to parse HTML with libxml {"parser":"libxml"} []
[2022-07-24 17:26:27] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call
 function "mb_regex_encoding" from namespace "Readability". Did you mean to call "\safe\mb_regex_encoding"?" at /var/www/wallabag2/vendor/j0k3r
/php-readability/src/Readability.php line 1403 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0
): Attempted to call function \"mb_regex_encoding\" from namespace \"Readability\".\nDid you mean to call \"\\safe\\mb_regex_encoding\"? at /var/www/wallabag2/vendor/j0k3r/php-readability/src/Readability.php:1403)"} []
```

**Comment by lapineige** [comment]
> same problem for me. I found the logfile in /var/www/wallabag2/var/logs/prod.log
But mine has grown to 1.5 GB in three years

That's strange, I thought we implemented logrotate to cut the log file into several piece and compress the archived ones…

**Comment by lapineige** [comment]
Linked issue in the forum https://forum.yunohost.org/t/mise-a-jour-impossible-de-wallabag/20485/7

**Comment by Schdefoon** [comment]
> > same problem for me. I found the logfile in /var/www/wallabag2/var/logs/prod.log
> > But mine has grown to 1.5 GB in three years
> 
> That's strange, I thought we implemented logrotate to cut the log file into several piece and compress the archived ones…

Was it implemented? I found [this issue ](https://github.com/YunoHost-Apps/wallabag2_ynh/issues/80)when looking for the storage location of the logfiles. The issue is already two years old but not closed. But never mind. For me it's a minor issue, too. No problem to solve the problem "by hand".

Best regards

**Comment by lapineige** [comment]
Good point :sweat_smile: 
Here it is : https://github.com/YunoHost-Apps/wallabag2_ynh/pull/144

**Comment by lapineige** [comment]
Could you all try this branch ?
yunohost app upgrade wallabag2 -u https://github.com/YunoHost-Apps/wallabag2_ynh/tree/fix_dependencies

It will upgrade to Wallabag 2.5 and hopefully fix some of your issue.

**Comment by gokceozan** [comment]
That solved the issue for me.

**Comment by deljones** [comment]
Many thanks indeed to the "gang" for making a new version of Wallbag available on YNH. I really appreciate it. Thank you very much...

dj

**Comment by lapineige** [comment]
Fixed by #139 #142
